### DISCUSSION TITLE
Tracking giada in downstream package ecosystem

### DISCUSSION THREAD

**Discussion by luzpaz** [monocasual/giada#666] [General]
### Current snapshot of downstream packages in the package ecosystem
<a href="https://repology.org/project/giada/versions">
    <img src="https://repology.org/badge/vertical-allrepos/giada.svg?columns=3" alt="Packaging status">
</a>

### TODO

- [ ] Chocolatey [windows binary] https://github.com/chocolatey-community/chocolatey-package-requests/issues/746
- [ ] Debian (bump downstream 'Debian Multimedia Maintainers')
- [ ] Fedora
- [ ] FreeBSD [downstream ticket](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=257416) pending on https://github.com/monocasual/giada/issues/581
- [ ] Gentoo
- [ ] Homebrew Cask [windows binary]
- [ ] OpenSUSE
- [ ] Raspbian  
- [ ] Solus (https://github.com/getsolus/packages/issues/616)
- [ ] Void (https://github.com/void-linux/void-packages/tree/master/srcpkgs/giada)
- [ ] Winget [windows binary] https://github.com/microsoft/winget-pkgs/issues/118874

**Comment by luzpaz** [comment]
Opened https://github.com/microsoft/winget-pkgs/issues/118874

**Reply by luzpaz** [reply]
Updated FreeBSD entry. Turns out it hinges on https://github.com/monocasual/giada/issues/581

**Reply by luzpaz** [reply]
bumped FreeBSD ticket

**Comment by gvnnz** [comment]
This is very cool, thanks @luzpaz

**Comment by luzpaz** [comment]
Downstream window binary repos are waiting for v0.26 to be released (https://github.com/chocolatey-community/chocolatey-package-requests/issues/746#issuecomment-1712567882). Is there an ETA on release date ?

**Reply by gvnnz** [reply]
v0.26 will be released pretty soon, definitely by the end of September. :crossed_fingers:

**Comment by luzpaz** [comment]
winget devs respond that they are unable to add giada for the following reason:  
> The package is released as a compressed ZIP file, with "loose executable" in it. For latest version of WinGet (currently v1.6), this kind of package is not supported.
>
> WinGet is not intended for this kind of packages, Scoop or Chocolatey may be better.

ref: https://github.com/microsoft/winget-pkgs/issues/118874#issuecomment-1753063734

**Comment by luzpaz** [comment]
Added request ticket to add Giada to Solus repo

**Reply by luzpaz** [reply]
There is a Solus maintainer interested :crossed_fingers:

**Reply by luzpaz** [reply]
bumped freebsd, solus, and chocolatey tickets. Waiting for responses.

**Comment by luzpaz** [comment]
Can someone bump Debian repos so we can get latest Giada in to the experimental repo at least ?
